lib/galaxy/datatypes/registry.py

```diff
@@ -21,6 +21,7 @@
 import binary
 import assembly
 import ngsindex
+import proteomics
```

Comment:
This line should no longer be needed as of maybe a couple releases ago - was it needed when testing this?